apps/web-client/src/lib/db/orders/__tests__/orders.test.ts

```diff
@@ -120,3 +127,37 @@ describe("Customer order tests", () => {
 		expect(db1Customers).toEqual([{ fullname: "Jane Doe", id: 1, email: "jane@example.com", deposit: 13.2 }]);
 	});
 });
+
+describe("Customer order status", () => {
+	let db: DB;
+	beforeEach(async () => {
+		db = await getRandomDb();
+
+		await createCustomerOrders(db);
+	});
+	it("can update the timestamp of when a customer order is placed (to supplier)", async () => {
+		const possibleOrderLines = await getPossibleSupplerOrderLines(db);
+
+		expect(possibleOrderLines).toStrictEqual([
+			{ supplier_id: 1, isbn: "1", quantity: 1 },
+			{ supplier_id: 1, isbn: "2", quantity: 1 },
+			{ supplier_id: 2, isbn: "3", quantity: 2 }
+		]);
+		// creating a supplier order should update the timestamp of placed
+		const newOrders = await createSupplierOrder(db, possibleOrderLines);
```

Comment:
I'm torn between suggesting to hardcode 
```
[
			{ supplier_id: 1, isbn: "1", quantity: 1 },
			{ supplier_id: 1, isbn: "2", quantity: 1 },
			{ supplier_id: 2, isbn: "3", quantity: 2 }
]
```

here or use `getPossibleSupplerOrderLines(db)` as a parameter. I do think that the assertion nedds to go though (as I stated in the comment above).